src/mui-theme-provider.jsx

```diff
@@ -0,0 +1,23 @@
+import {Component, PropTypes} from 'react';
+
+class ThemeProvider extends Component {
+  getChildContext() {
+    return {
+      muiTheme: this.props.muiTheme,
+    };
+  }
+  render() {
+    return this.props.children;
+  }
+}
+
+ThemeProvider.propTypes = {
+  children: PropTypes.element,
+  muiTheme: PropTypes.object,
+};
+
+ThemeProvider.childContextTypes = {
```

Comment:
When I tried making them static members, the eslinter doesn't detect that `muiTheme` and `children` as propTypes, and produces an error...I noticed we're not using the latest versions of some eslint related packages. I can investigate a bit...